script.js

```diff
@@ -1,4 +1,181 @@
+var reverseModeStatus = false;
+var userWinCount = 0;
+var computerWinCount = 0;
+var drawCount = 0;
+var userName = "";
+
+var RandomSPS = function () {
+  randomDecimal = Math.random();
+  randomInteger0to2 = Math.floor(randomDecimal * 3);
+  RandomInterger1to3 = randomInteger0to2 + 1;
+  if (RandomInterger1to3 == 1) {
+    computerChoiceMsg = "The computer chose Scissors!";
+    return "Scissors";
+  }
+  if (RandomInterger1to3 == 2) {
+    computerChoiceMsg = "The computer chose Paper!";
+    return "Paper";
+  }
+  if (RandomInterger1to3 == 3) {
+    computerChoiceMsg = "The computer chose Stone!";
+    return "Stone";
+  }
+};
```

Comment:
Best not to have names that begin with Caps. Should practice camelCase naming

E.g. this should be named randomSps